owasp-top10-2017-apps/a5/ecommerce-api/docs/ATTACK.md

```diff
@@ -0,0 +1,45 @@
+# Attack Narrative - Ecommerce API
+The main goal of this document is to describe how a malicious user could exploit a lack of access control to obtain tokens that he should not have access to.
+
+If you don't know [secDevLabs] or this [intended vulnerable web application][2] yet, you should check them before reading this narrative.
+
+---
+## 👀
+
+In order to better understand how this API works, two users were created, `user1` and `user2`, as shown below: 
```

Comment:
What do you think of changing `two users were created, user1 and user2, as shown below` into `two users, user1 and user2, were created using the following curl commands:` ?

Reply:
Thanks @rafaveira3 ! I think it does looks better your way! 👍